keyboardvisibilityevent/src/main/java/net/yslibrary/android/keyboardvisibilityevent/KeyboardVisibilityEvent.java

```diff
@@ -21,7 +21,7 @@
      * @param activity Activity
      * @param listener KeyboardVisibilityEventListener
      */
-    public static void setEventListener(final Activity activity,
+    public static Deregister setEventListener(final Activity activity,
```

Comment:
My bad, to meet the naming consistency, this should be renamed to `registerEventListener`